api/grpcserver/smesher_service.go

```diff
@@ -68,20 +69,20 @@ func (s SmesherService) StartSmeshing(ctx context.Context, in *pb.StartSmeshingR
 		return nil, status.Error(codes.InvalidArgument, "`Opts.MaxFileSize` must be provided")
 	}
 
-	opts := activation.PostSetupOpts{
-		DataDir:           in.Opts.DataDir,
-		NumUnits:          in.Opts.NumUnits,
-		MaxFileSize:       in.Opts.MaxFileSize,
-		ComputeProviderID: int(in.Opts.ComputeProviderId),
-		Throttle:          in.Opts.Throttle,
-		Scrypt:            config.DefaultLabelParams(),
```

Comment:
Alternatively I'd be OK to set `ComputeBatchSize` to `config.DefaultComputeBatchSize` here as well and we give users the option to tinker with this value in the future.

Reply:
I previously did that, but I don't think setting `ComputeBatchSize` to `config.DefaultComputeBatchSize` here is a good approach, because that would ignore any  `ComputeBatchSize` set in the initial smeshing config used when starting the node.

Reply:
Alright, but can we stick with `opts` instead of `optsCopy`? It's not a copy of the opts, its the opts provided by the client (with some defaulting to what the node was started with).

Reply:
Are you suggesting rename `optsCopy -> opts` ?